src/PiCraft.jl

```diff
@@ -98,7 +101,7 @@ function mc_send(cmd, output=true)
     if minecraftWorld.s.status == Base.StatusInit || minecraftWorld.s.status == Base.StatusUninit
         error("Connection to Minecraft World is not initialised. Use `PiCraft.connectToWorld()` first.")
     end
-    println("Command: $cmd")
+    #println("Command: $cmd")
     write(minecraftWorld.s, cmd*"\n")
```

Comment:
Certain scripts may have thousands of blocks to be added. Better to not litter the stream. ?

Reply:
Yeah, this was only meant for temporary debugging.